openlibrary/templates/site/body.html

```diff
@@ -29,11 +29,16 @@
       $for flash in get_flash_messages():
         <div class="$flash.type"><span>$flash.message</span></div>
       </div>
-      $# Announcement banner will only be rendered if announcement and storage_key variables are set.
-      $# Be sure to escape any single quotes inside of the announcement HTML string.
-      $ announcement = ''
+      $ learn_more_copy = _('Learn More')
+      $ learn_more_link = '<a style="color: white;" href="https://blog.archive.org">%s</a>' % learn_more_copy
+
+      $ sign_copy = _('Sign Letter')
+      $ sign_btn = '<a class="btn primary" style="display: inline-block;" href="https://change.org/LetReadersRead">%s</a>' % sign_copy
+
+      $ announcement = '&#9888; ' + _('Urge publishers to restore access to 500,000 wrongly removed library books: %(sign_btn)s - %(learn_more_link)s', sign_btn=sign_btn, learn_more_link=learn_more_link)
+
       $# Include date in cookie name to avoid name collisions (something like '{prefix}{YY}{MM}{DD}' would work well)
-      $ cookie_name = ''
+      $ cookie_name = 'j28'
       $if announcement and cookie_name:
         $:render_template('site/banner', announcement, cookie_name, cookie_duration_days=30)
```

Comment:
Related to #8892, I think cookie dismiss code may not be working if patrons are logged out because ctx.user fails so we may need another way to access the cookie than `ctx.user.has_cookie`
e.g. `web.cookies().get(cookie_name)`
![image](https://github.com/internetarchive/openlibrary/assets/978325/75c00eb9-5e10-43a5-98b4-3f2b0afade62)